### PR TITLE
Use safe escape for markdown blocks

### PR DESCRIPTION
**Pull request by zerolab**
renders the markdown block value with the `|safe` filter.

Required to make code blocks readable (e.g https://torchbox.com/blog/apache-solr-search-solr-47/)

### CODE REVIEW

**Comment by zerolab** [comment]
Needs work

File: tbx/core/templates/torchbox/includes/streamfield.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
                     </div>
                 </div>
             {% elif child.block_type == 'raw_html' %}
-                {{ child.value|safe }}
```
Comment (nimasmi) [inline tbx/core/templates/torchbox/includes/streamfield.html]: Shouldn't this`|safe` remain?

**Comment by zerolab** [comment]
It made perfect sense.. At the time. That crazy @zerolab needs punishing. Or something.
Well, `spaceless` should take in consideration `<pre>` tags. 

Rebased to contain the proper fixes

**Comment by zerolab** [comment]
@nimasmi I think you were looking for this earlier today

**Comment by nimasmi** [comment]
Fancy that. It was a rabbit hole, I tell you. Thanks, Dan.

**Comment by zerolab** [comment]
@victoriachan @martycoote this is a nice quick one that would make code block readable. May need a rebase / tweaks.

**Comment by zerolab** [comment]
superseded by https://github.com/torchbox/torchbox-frontend/pull/90

**Pull request closed without merging**